src/nats.h

```diff
@@ -27,13 +27,16 @@ extern "C" {
 #include "status.h"
 #include "version.h"
 
-#if defined(NATS_HAS_TLS)
+#ifdef NATS_WITH_EXPERIMENTAL
+
+// natsOptions_SetSSLVerificationCallback 
```

Comment:
Remove?

Reply:
This was in case we ever have more than a handful EXPERIMENTAL APIs, but they prob. wouldn't be grouped together anyway; will remove when I get back soon